index.js

```diff
@@ -38,7 +39,10 @@ app.get("*", function (req, res) {
 	res.status(404).render("notfound", {
```

Comment:
It will be helpful to format this file before the next push so that the code segment is aligned. In this code review system, line 39 is appearing over-indented from the reviewer's perspective.